src/coord/rmr/cluster_topology.c

```diff
@@ -35,6 +35,19 @@ void MRClusterTopology_AddShard(MRClusterTopology *topo, MRClusterShard *sh) {
   topo->shards[topo->numShards++] = *sh;
 }
 
+void MRClusterTopology_SortShards(MRClusterTopology *topo) {
```

Comment:
```suggestion
void MRClusterTopology_SortShardsByPort(MRClusterTopology *topo) {
```

perhaps it is a clearer naming